chainer/gradient_check.py

```diff
@@ -122,13 +122,18 @@ def check_backward(func, x_data, y_grad, params=(),
                    eps=1e-3, atol=1e-5, rtol=1e-4, no_grads=None, dtype=None):
     """Test backward procedure of a given function.
 
-    This function automatically checks backward-process of a given function.
-    For example, when you have a :class:`~chainer.Function` class ``MyFunc``,
-    that gets two arguments and returns one value, you can make its test like
-    this::
+    This function automatically checks the backward-process of a given function
+    to ensure that the computed gradients are approximately correct.
+    For example, assuming you've defined a :class:`~chainer.FunctionNode` class
+    ``MyFunc``, that takes two arguments and returns one value, you can wrap
+    it in a ordinary function and check its gradient computations as follows::
 
     >> def test_my_func(self):
-    >>   func = MyFunc()
+    >>
```

Comment:
I have a question about it. The original one does not work correctly, right? It is better to test code snippets in docstring with doctest.

Reply:
No I don't think it worked. I agree. Maybe this is better done in a separate PR since this one already affect quite a large number of files.

Reply:
OK, I agree with you.